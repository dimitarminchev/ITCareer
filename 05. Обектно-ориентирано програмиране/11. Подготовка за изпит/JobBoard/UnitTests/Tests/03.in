AddCategory I
AddCategory It
AddJobOffer It TechSupport MyTech 1500 remote True
AddJobOffer It OnSiteTechSupport MyTech 1500 onsite Burgas
AddJobOffer It Tester YourTest 2000 onsite Varna
End