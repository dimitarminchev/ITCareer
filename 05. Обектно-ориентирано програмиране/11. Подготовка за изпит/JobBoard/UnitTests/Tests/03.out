Name should be between 2 and 40 characters!
Created Category It!
Created JobOffer TechSupport in Category It!
Created JobOffer OnSiteTechSupport in Category It!
Created JobOffer Tester in Category It!